docs/contributors/guide/cherry-pick/cherry-pick.md

```diff
@@ -0,0 +1,123 @@
+<!-- Lots of ideas references from https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/cherry-picks.md, Thank!-->
+
+# Overview
+
+This document explains how cherry picks are managed on release branches within the `pingcap/tiflow` repository. A common
+use case for this task is backporting PRs from master to release branches.
+
+- [Prerequisites](#prerequisites)
+- [What Kind of PRs are Good for Cherry Picks](#what-kind-of-prs-are-good-for-cherry-picks)
+- [Initiate a Cherry Pick](#initiate-a-cherry-pick)
+- [Cherry Pick Review](#cherry-pick-review)
+- [Searching for Cherry Picks](#searching-for-cherry-picks)
+- [Troubleshooting Cherry Picks](#troubleshooting-cherry-picks)
+- [Batch Merging Cherry Picks](#batch-merging-cherry-picks)
+
+## Prerequisites
+
+- A pull request merged against the `master` branch.
+- The release branch exists (example: [`release-5.0`](https://github.com/pingcap/tiflow/tree/release-5.0))
+
+## What Kind of PRs are Good for Cherry Picks
```

Comment:
cherry pick standard should be  concerned with the release version .

Reply:
Do you mean the maintenance cycle? But at the moment we don't seem to be able to say what the maintenance intervals are. So I didn't add it.

Reply:
less fixs should be cherry picked to the older release version